dist/flags.py

```diff
@@ -41,6 +41,12 @@ def flag_declare(name):
                         " GENERATION STOP 32", file=sys.stderr)
                     sys.exit(1)
                 end = int(m.group(0))
+
+                poweroftwo = (end != 0) and ((end & (end-1)) == 0)
+                if not poweroftwo and end != 12:
+                    print(name + ": line " + str(lcnt) + ", the stop value " + str(end) +
+                    " is not a power of 2", file=sys.stderr)
```

Comment:
Did you run this on current develop alone and did it find/flag `btmem.h`?

Reply:
Yes, when I ran the script (`s_all`) without this change on the latest develop then it complained `btmem.h` of using 9 as a stop value.